### DISCUSSION TITLE
OrientDB jnr-posix Licensing

### DISCUSSION THREAD

**Discussion by abhaychaware** [orientechnologies/orientdb#10252] [General]
orientDb-graphdb-3.2.29  itself is opensource and licensed under Apache 2.0, but one of its dependencies jnr-posix-3.1.15 is licensed under multiple licenses including GPL.

https://github.com/jnr/jnr-posix/blob/jnr-posix-3.1.15/LICENSE.txt

Since I am a consumer of orientDb-graphdb-3.2.29 which is Apache license, do I need to worry about GPL licensing of jnr-posix-3.1.15?

**Comment by tglman** [comment]
Hi,

As is written in the license file you linked 

```
  Eclipse Public License version 2.0
    OR
  GNU General Public License version 2
    OR
  GNU Lesser General Public License version 2.1
```

Eclipse Public License is compatible with Apache 2

**Reply by abhaychaware** [reply]
Thanks for the response.  I am using this jar orientDb-graphdb-3.2.29 ( Apache2.0 license ) in my **commercial** product.  Out of the three license opions provided for this jnr-posix-3.1.15 library, can I simply "choose" the "Eclipse Public License version 2.0" license for this library, as I am using it in a etirely as-is/non-modified way?

**Reply by tglman** [reply]
I'm not a lawyer myself, and you should be speaking with one if you have doubts for your **commercial** product.

What I can tell you is that from our how evaluations this do no create issue with Apache license that's why is in the dependencies.

**Reply by abhaychaware** [reply]
Okay, thank you for your response @tglman.
